src/models/Connection.ts

```diff
@@ -3,7 +3,7 @@ export type Connection = {
   isConnected: boolean;
 };
 
-export function setConnectionState(
+export function createConnection(
```

Comment:
we said to change the name to setConnectionState, because we were not always starting the connection, but sometimes it was to change the state.